cluster/test-deploy/azure/vars.yaml

```diff
@@ -0,0 +1,34 @@
+---
+openshift_azure_clusterid: magentaci
+openshift_azure_resource_location: eastus
+
+openshift_deployment_type: openshift-enterprise
+
+openshift_azure_vm_admin_username: openshift
+ansible_ssh_user: "{{ openshift_azure_vm_admin_username }}"
```

Comment:
Why isn't this in the core playbooks?  It's image specific?  Seems like we'd want the cloud provider consistent value.